_posts/2018-08-26-be-productive-with-org-mode.md

```diff
@@ -0,0 +1,175 @@
+- some items have priorities - A, B, C.
+- links can be internal (`[[file:~/...]`)
+
+#### Capture templates
+
+As described in Org-mode's documentation, capture lets you quickly store notes with little interruption of your
+workflow.
+
+I configured several capture templates which help me to quickly create notes about things that I want to remember.
+
+```lisp
+  (setq org-capture-templates
+        '(("t" "Todo" entry (file+headline "~/Dropbox/org/todo.org" "Todo soon")
+           "* TODO %? \n  %^t")
+          ("i" "Idea" entry (file+headline "~/Dropbox/org/ideas.org" "Ideas")
+           "* %? \n %U")
+          ("e" "Tweak" entry (file+headline "~/Dropbox/org/tweaks.org" "Tweaks")
+           "* %? \n %U")
+          ("l" "Learn" entry (file+headline "~/Dropbox/org/learn.org" "Learn")
+           "* %? \n")
+          ("w" "Work note" entry (file+headline "~/Dropbox/org/work.org" "Work")
+           "* %? \n")
+          ("m" "Check movie" entry (file+headline "~/Dropbox/org/check.org" "Movies")
+           "* %? %^g")
+          ("n" "Check book" entry (file+headline "~/Dropbox/org/check.org" "Books")
+           "* %^{book name} by %^{author} %^g")))
+```
+
+For a book note I should add its name and its author, for a movie note I should add tags etc.
+
+### Planning
+
+Another great feature of Org-mode is that you can use it as a day planner. Let's see an example of one of my days:
+
+![schedule](https://i.imgur.com/z5HpuB0.png)
+
+I didn't give a lot of thought to this example, it's my real file for today. It doesn't look like much but it helps to spend your time on things that important to you and fight with procrastination.
+
+#### Habits
+
+From Org mode's documentation, Org has the ability to track the consistency of a special category of
+TODOs, called “habits”. I use this feature along with day planning when I want to create new habits:
+
+![habits](https://i.imgur.com/YJIp3d0.png)
+
+As you can see currently I'm trying to wake early every day and workout once in two days. Also, it helped to start reading books every day.
+
+#### Agenda views
+
+Last but not least I use agenda views. Todo items can be scattered throughout different files (in my case daily plan and habits are in separate files), agenda views give an overview of all todo items:
+
+![agenda](https://i.imgur.com/CKX9BL9.png)
+
+### More Org mode features
+
+- Smartphone apps ([Android](https://play.google.com/store/apps/details?id=com.orgzly&hl=en), [ios](https://itunes.apple.com/app/id1238649962))
+
+- [Exporting Org mode files into different formats](https://orgmode.org/manual/Exporting.html) (html, markdown, pdf, latex etc)
+
+- [Tracking Finances](https://orgmode.org/worg/org-tutorials/weaving-a-budget.html) with [ledger](https://github.com/ledger/ledger-mode)
+
+### Conclusion
+
+In this post, I described a small subset of Org-mode's extensive functionality that helps me be productive every day, spend time on things that important to me.
```

Comment:
I think it may be better to say `spending time on things....` (from "spend" to "spending").